subsys/bluetooth/host/att.c

```diff
@@ -3832,6 +3832,27 @@ uint16_t bt_att_get_mtu(struct bt_conn *conn)
 	return mtu;
 }
 
+uint16_t bt_att_get_uatt_mtu(struct bt_conn *conn)
+{
+	struct bt_att_chan *chan, *tmp;
+	struct bt_att *att;
+
+	att = att_get(conn);
+	if (!att) {
+		return 0;
+	}
+
+	SYS_SLIST_FOR_EACH_CONTAINER_SAFE(&att->chans, chan, tmp, node) {
+		if (!bt_att_is_enhanced(chan)) {
+			return bt_att_mtu(chan);
+		}
+	}
+
+	/** There must be an UATT channel per connection */
+	k_oops();
```

Comment:
It the comment above is an assertion, then it's correct to put an `__ASSERT_NO_MSG(false)` here.

OTOH, if the comment above is something you are unsure about (I don't blame you -- I don't think it's obvious e.g. during connection teardown), then it's better to try recovering. Returning `0`, if we document that, or the default ATT_MTU value may be better.

@jori-nordic What do you think?